spec/lib/payment_calculator/turnip_features/ecf/uplift_payment.feature

```diff
@@ -0,0 +1,8 @@
+@ecf
+Feature: ECF contract calculations
+
+  Scenario: Uplift payment
+    Given An uplift per participant of £100
+    And there are 100 sparsity and 200 pupil premium participants who have started that are eligible for the uplift payment
```

Comment:
This would make more sense to me as two steps

```suggestion
    And there are 100 sparsity participants who have started that are eligible for the uplift payment
    And there are 200 pupil premium participants who have started that are eligible for the uplift payment
```

Reply:
Done in a5f3779